privacyidea/api/validate.py

```diff
@@ -207,6 +207,7 @@ def offlinerefill():
 @postpolicy(check_tokentype, request=request)
 @postpolicy(check_serial, request=request)
 @postpolicy(autoassign, request=request)
+@postpolicy(preferred_client_mode, request=request)
```

Comment:
I am not sure if we also should add this to triggerchallenge!?
Maybe we need to discuss this with @nilsbehlen